projects/core/src/user/store/effects/user-register.effect.ts

```diff
@@ -12,19 +12,15 @@ import { UserActions } from '../actions/index';
 export class UserRegisterEffects {
   @Effect()
   registerUser$: Observable<
-    UserActions.UserRegisterOrRemoveAction | AuthActions.LoadUserToken
+    UserActions.UserRegisterOrRemoveAction
   > = this.actions$.pipe(
     ofType(UserActions.REGISTER_USER),
     map((action: UserActions.RegisterUser) => action.payload),
     mergeMap((user: UserSignUp) =>
       this.userConnector.register(user).pipe(
-        switchMap(_result => [
```

Comment:
Dispatching `LoadUserToken` should also be conditional on `featureLevel` flag. We can't just remove it.

Reply:
I'm not exactly sure if we can add featureLevel to effect